tests/FunctionalTests/Command/ResetPasswordRemoveExpiredCommandTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/*
+ * This file is part of the SymfonyCasts ResetPasswordBundle package.
+ * Copyright (c) SymfonyCasts <https://symfonycasts.com/>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SymfonyCasts\Bundle\ResetPassword\Tests\FunctionalTests\Command;
+
+use Doctrine\Bundle\DoctrineBundle\Registry;
+use Doctrine\ORM\Tools\SchemaTool;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\CommandLoader\CommandLoaderInterface;
+use Symfony\Component\Console\Tester\CommandTester;
+use SymfonyCasts\Bundle\ResetPassword\Tests\Fixtures\AbstractResetPasswordTestKernel;
+use SymfonyCasts\Bundle\ResetPassword\Tests\Fixtures\Entity\ResetPasswordTestFixtureRequest;
+
+/**
+ * @author Jesse Rushlow <jr@rushlow.dev>
+ * @author Ryan Weaver   <ryan@symfonycasts.com>
+ *
+ * @internal
+ */
+final class ResetPasswordRemoveExpiredCommandTest extends TestCase
+{
+    /**
+     * @var \Doctrine\Common\Persistence\ObjectManager|object
```

Comment:
This ns should be fixed in a separate PR. Related to #75 

I have to bump the `doctrine/doctrine-bundle` version in dev to minimum of `2.0.3` to utilize the new `Doctrine\Persistence\...` namespace.